app/code/Magento/Inventory/Setup/InstallData.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Setup;
+
+use Magento\Framework\Setup\InstallDataInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\Api\DataObjectHelper;
+
+/**
+ * Class InstallData
+ */
+class InstallData implements InstallDataInterface
+{
+    /**
+     * @var SourceRepositoryInterface $stockRepository
+     */
+    private $sourceRepository;
+
+    /**
+     * @var ObjectManagerInterface
+     */
+    private $objectManager;
+
+    /**
+     * @var DataObjectHelper
+     */
+    private $dataObjectHelper;
+
+    /**
+     * @param SourceRepositoryInterface $sourceRepository
+     * @param ObjectManagerInterface    $objectManager
+     * @param DataObjectHelper          $dataObjectHelper
+     */
+    public function __construct(
+        SourceRepositoryInterface $sourceRepository,
+        ObjectManagerInterface $objectManager,
+        DataObjectHelper $dataObjectHelper
+    ) {
+        $this->sourceRepository = $sourceRepository;
+        $this->objectManager    = $objectManager;
```

Comment:
:interrobang: :boom: Pls don't use the object manager inject the **SourceInterface**